flow/connectors/mongo/mongo.go

```diff
@@ -47,6 +47,7 @@ func NewMongoConnector(ctx context.Context, config *protos.MongoConfig) (*MongoC
 		SetAppName("PeerDB Mongo Connector").
 		SetReadPreference(readpref.Primary()).
 		SetCompressors([]string{"zstd", "snappy"}).
+		SetDirect(true).
```

Comment:
I think this only works for specifying a single host in connection uri, which is fine for now. 
Later I suspect we'll need to support multiple hosts or srv uri